scripts/mail-all-activity.py

```diff
@@ -1,181 +0,0 @@
-#!/usr/bin/env python
```

Comment:
1. Would a `.sh` script that sets the required envvars on the production server work?
2. From where did you send the email last time? The `mail` command used in that old script is still supported in this new one: https://github.com/swcarpentry/amy/pull/629/files#diff-60f31abade2bde4c5ee02a99d9781df4R96